pkg/whitesource/configHelper.go

```diff
@@ -87,6 +83,10 @@ func (c *ConfigOptions) updateConfig(originalConfig *map[string]string) map[stri
 			}
 		}
 	}
+	for k, v := range *originalConfig {
```

Comment:
So with this logic irrespective of value `force `or `append ` , original config values will be replaced in the newconfig ? 
Is it overwriting the existing enforcing and appending logic ?